src/components/MentionSuggestions.tsx

```diff
@@ -1,73 +1,71 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
-import _ from 'underscore';
 import getStyledTextArray from '@libs/GetStyledTextArray';
 import styles from '@styles/styles';
 import * as StyleUtils from '@styles/StyleUtils';
 import themeColors from '@styles/themes/default';
 import CONST from '@src/CONST';
 import AutoCompleteSuggestions from './AutoCompleteSuggestions';
 import Avatar from './Avatar';
-import avatarPropTypes from './avatarPropTypes';
 import Text from './Text';
 
-const propTypes = {
-    /** The index of the highlighted mention */
-    highlightedMentionIndex: PropTypes.number,
+// TODO: remove when avatarPropTypes will be migrated to TS
```

Comment:
should we keep this comment?

Reply:
also I belive that its not the correct type.
the `source` prop should be a type of `string | React.FC<SvgProps>`

Reply:
It was added according to the propTypes file 

<img width="615" alt="image" src="https://github.com/Expensify/App/assets/23215729/0cd821f6-5e51-4722-82ae-81cf2a3ed8a4">

Reply:
PropTypes aren't always correct, you have to check the usage

Reply:
I believe that further extensions should be part of Avatar component migration / propTypess migration

Reply:
You can also debug it a little, console.log what is the type actually. Have you tried it?

Reply:
I didn't see messages in this thread except the first two, but now it looks like dialogue :joy:

Reply:
I believed before that a deep investigation for types that are not related directly should be part of the directly related component migration (Avatar migration in this case)